garrysmod/lua/vgui/dlabelurl.lua

```diff
@@ -56,14 +65,34 @@ function PANEL:GetColor()
 
 end
 
+--[[---------------------------------------------------------
+	SetFont
+-----------------------------------------------------------]]
+function PANEL:SetFont( strFont )

```

Comment:
There's no need for this, this method is inherited from DLabel ( Which is parent of DButton ).